README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
+
+The main Sass (SCSS) source file is here: 
+
+```
+node_modules/uswds/src/stylesheets/all.scss
+```
+
+The non-minified CSS that’s been precompiled is here:
```

Comment:
I think @juliaelman meant to add the minified version in addition to the non-minified version here. Is that correct? I don't think she mentioned updating the example HTML to minified but just for this section.